src/tabrmd-defaults.h

```diff
@@ -16,7 +16,7 @@
 #define TABRMD_ENTROPY_SRC_DEFAULT "/dev/urandom"
 #define TABRMD_SESSIONS_MAX_DEFAULT 4
 #define TABRMD_SESSIONS_MAX 64
-#define TABRMD_TCTI_CONF_DEFAULT "device:/dev/tpm0"
+#define TABRMD_TCTI_CONF_DEFAULT ((!access("/dev/tcm0", F_OK)) ? ("device:/dev/tcm0") : ("device:/dev/tpm0"))
```

Comment:
No, this is configured code so we don;t want to dynamically dispatch off it, and I wonder why we don't just use TCTILdr with a NULL prefix since that has our loader logic in it, and add tcm0 as an option there.